iree/compiler/Dialect/VM/Conversion/StandardToVM/ConvertStandardToVM.cpp

```diff
@@ -793,6 +817,8 @@ void populateStandardToVMPatterns(MLIRContext *context,
                   ModuleOpConversion, FuncOpConversion, ReturnOpConversion,
                   SelectOpConversion>(typeConverter, context);
 
+  patterns.insert<FloorDivSIOpConversion>(typeConverter, context);
```

Comment:
This should also have a test case added to https://github.com/google/iree/blob/main/iree/compiler/Dialect/VM/Conversion/StandardToVM/test/arithmetic_ops.mlir